frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/vm-settings-tab-state-update.ts

```diff
@@ -110,11 +110,9 @@ const provisioningSourceUpdater = ({ id, prevState, dispatch, getState }: Update
   dispatch(
     vmWizardInternalActions[InternalActionType.UpdateVmSettings](id, {
       [VMSettingsField.CONTAINER_IMAGE]: {
```

Comment:
the fields were kept required so it would be apparent from redux which fields have to be filled.

for this special use case `skipValidation` was introduced into `vm-settings-tab-initial-state`.

Please revert this change or give a good reason why it shouldn't be visible in redux and skipValidation should then be removed from initial-state?

Reply:
> the fields were kept required so it would be apparent from redux which fields have to be filled.

this fields are are not required in the user view, a user can leave them empty and continue filling up the form.
the redux viewer should reflect that.

> or give a good reason why it shouldn't be visible in redux and skipValidation should then be removed from initial-state?

This fields are optional, users can fill this information later in the storage tab, we should remove the skipValidation, I can make a folow up PR to fix that.

Reply:
+1 let's remove the skipValidation then